src/core/services/theming/theming.spec.js

```diff
@@ -752,8 +752,10 @@ describe('$mdTheming service', function() {
   it('supports registering theme on the fly', inject(function ($mdTheming) {
     expect($mdTheming.THEMES.hasOwnProperty('test')).toBeFalsy();
 
+    // optional <color>Hue option for each color, as in normal theme definition.
     $mdTheming.defineTheme('test', {
       primary: 'red',
+      primaryHue: 300,
```

Comment:
We should probably have a test that verifies that the correct hue was defined/used.

Reply:
I have little experience with writing tests. I'll look into it and see what I can do.